lib/constraintMatcher.js

```diff
@@ -411,16 +464,20 @@ var lang = {
 var matcherCount = 0;
 var toJs = exports.toJs = function (rule, scope, alias, equality, wrap) {
     /*jshint evil:true*/
-    var js = lang.parse(rule);
-    scope = scope || {};
-    var vars = lang.getIdentifiers(rule);
-    var closureVars = ["var indexOf = definedFuncs.indexOf; var hasOwnProperty = Object.prototype.hasOwnProperty;"], funcVars = [];
-    extd(vars).filter(function (v) {
+	var  closureVars = ["var indexOf = definedFuncs.indexOf; var hasOwnProperty = Object.prototype.hasOwnProperty;"], 
```

Comment:
not clear to me other than tightening the var decl what the formatting issue is ?